lib/livebook_web/live/settings_live/file_systems_component.ex

```diff
@@ -55,4 +60,45 @@ defmodule LivebookWeb.SettingsLive.FileSystemsComponent do
     <.labeled_text label="Bucket URL"><%= @file_system.bucket_url %></.labeled_text>
     """
   end
+
+  defp default_file_system(
+         %{id: file_system_id, default_file_system: default_file_system} = assigns
+       ) do
+    case Ets.fetch(:file_system, "default_file_system") do
```

Comment:
We should keep default file system (actually just the id should be enough) in assigns in the LV and pass it down. This way we don't need to fetch on every render.